debugging/tasks/clear-maintenance.js

```diff
@@ -0,0 +1,62 @@
+/*jshint globalstrict:false, strict:false, sub: true */
+/*global ARGUMENTS, print, arango, db */
+exports.name = "clear-maintenance";
+exports.group= "cleanup tasks";
+exports.args = [];
+
+exports.args_arangosh = " --server.endpoint LEADER-AGENT";
+exports.description = "Checks the state of the supervision";
+exports.selfTests = ["arango", "db", "agencyConnection"];
+exports.requires = "3.3.23 - 3.6.99";
+exports.info = `
+Checks the state of the supervision.
+`;
+
+exports.run = function(extra, args) {
+  const helper = require('../helper.js');
+
+  // imports
+  const _ = require('underscore');
+
+  // get an agency dump
+  const dump = helper.getAgencyDumpFromObjectOrAgency(undefined);
+  const state = dump.arango.Supervision.State;
+  const maintenance = dump.arango.Supervision.Maintenance;
+  const create = dump.arango.Target.HotBackup.Create;
```

Comment:
In the case some subpath does not exist, this will throw an exception.

Reply:
@maierlars added safety check for Target and HotBackup